Src/AutoFixture.xUnit.net2.UnitTest/TypeWithCustomizationAttributes.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Ploeh.TestTypeFoundation;
+
+namespace Ploeh.AutoFixture.Xunit2.UnitTest
+{
+    internal class TypeWithCustomizationAttributes
+    {
+        public void CreateWithFrozenAndGreedy([Frozen, Greedy] ConcreteType sut)
+        {
+        }
+
+        public void CreateWithGreedyAndFrozen([Greedy, Frozen] ConcreteType sut)
+        {
+        }
```

Comment:
Wouldn't it be safer if we tested with more than 2 attributes? For example Frozen, Greedy, and FavorArrays attributes.